CHANGELOG.next.asciidoc

```diff
@@ -94,6 +94,7 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - Fixed a bug where `event.duration` could be missing from an event on Windows systems due to low-resolution clock. {pull}44440[44440]
 - Add check for http error codes in the Metricbeat's Prometheus query submodule {pull}44493[44493]
 - Sanitize error messages in Fetch method of SQL module {pull}44577[44577]
+- Fixed histogram values of zero are filtered out on non-amd64 platforms for openmetrics and prometheus {pull}44750[44750]
```

Comment:
You're added that in the breaking changes section. The bug fixes for Metricbeat start on L257

Reply:
thx @belimawr I moved it to the right section